tests/containers/secret.pm

```diff
@@ -20,6 +20,10 @@ sub run {
     my ($self, $args) = @_;
     my $output = '';
 
+    my $podman_version = get_podman_version();
+    # Skip this module on podman < 3.1.0
```

Comment:
the command was added in 4.5.0, I think we should change the condition here
```
    # "podman secret exists" was added to podman 4.5.0 according to
    # https://github.com/containers/podman/blob/main/RELEASE_NOTES.md#450
```

Reply:
> the command was added in 4.5.0, I think we should change the condition here

podman secret is much older.